web/cypress/integration/route/create-edit-route-with-redirect-plugin.spec.js

```diff
@@ -51,7 +51,6 @@ context('Create Edit and Delete Route with redirect plugin', () => {
   it('should create route with custom redirect plugin', function () {
     cy.visit('/');
     cy.contains('Route').click();
-    cy.get(selector.empty).should('be.visible');
```

Comment:
why remove this line, this is to ensure that there is no other dirty data when the route is created